generated/nidaqmx/_library_interpreter.py

```diff
@@ -2410,8 +2408,7 @@ def get_chan_attribute_bool(self, task, channel, attribute):
         cfunc = lib_importer.cdll.DAQmxGetChanAttribute
         with cfunc.arglock:
             cfunc.argtypes = [
-                lib_importer.task_handle, ctypes_byte_str, ctypes.c_int32,
-                ctypes.POINTER(c_bool32)]
+                lib_importer.task_handle, ctypes_byte_str, ctypes.c_int32]
```

Comment:
Duplicating the argtypes setup for one C function between multiple Python methods makes it possible for the Python methods to set the C function's argtypes inconsistently. 

It would be possible to ensure that this never happens by refactoring the codegen to generate shared helper methods. However, I don't think it's critical to do this refactor, as long as we have test coverage for all of the Python get/set/reset methods.

Looking at the test coverage for `_library_interpreter.py`, there are some gaps. (To view the test coverage, run `poetry run pytest -v --cov generated/nidaqmx` and `poetry run coverage html`, then open `htmlcov/index.html`.)

Addressing this is outside the scope of this PR, so please create a task.

Reply:
Sure Thanks for providing the commands for running test coverage. 
I have created a task **[Task 2388233](https://dev.azure.com/ni/DevCentral/_workitems/edit/2388233): Increase test coverage for _library_interpreter.py** to cover the test coverage for _library_interpreter.py. 
Should this task scope for only get/set/reset functions or is it better to scope it for coverage of library interpreter on whole?

Reply:
I think we should evaluate the scope for the interpreter as a whole, but that doesn't necessarily mean 100% coverage. We can discuss it more later.

Reply:
@bkeryan , adding coverage for interpreter would require adding test cases for almost all of the nidaqmx which we did not plan when putting down the HLD. We are getting closer to the dev exit, do you think the new test cases addition are in the critical path of the DAQmx epic? 

Could we have this work item as tech debt out of the Epic scope. What do you think? We could also take this discussion over teams.